bip-0387.mediawiki

```diff
@@ -0,0 +1,101 @@
+<pre>
+  BIP: 387
+  Layer: Applications
+  Title: Tapscript Multisig Output Script Descriptors
+  Author: Pieter Wuille <pieter@wuille.net>
+          Ava Chow <me@achow101.com>
+  Comments-Summary: No comments yet.
+  Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-0387
+  Status: Draft
+  Type: Informational
+  Created: 2024-04-17
+  License: BSD-2-Clause
+</pre>
+
+==Abstract==
+
+This document specifies <tt>multi_a()</tt> and <tt>sortedmulti_a()</tt> output script descriptors.
```

Comment:
To ask the dumb question, what is `a`? `add`? 

`csa` would be clearer IMO.

Reply:
The meaning of `a` is left for the reader to decide :p 

(I don't think it has any meaning, just something to distinguish from `multi`. It's too late to change it since people are already using this.)